singularity-eos/eos/eos_spiner.hpp

```diff
@@ -14,1997 +14,12 @@
 
 #ifndef _SINGULARITY_EOS_EOS_EOS_SPINER_HPP_
 #define _SINGULARITY_EOS_EOS_EOS_SPINER_HPP_
```

Comment:
I think you want this new `eos_spiner.hpp` file to include the two new files, so that we can load them both simultaneously easily when desired.